api/http/handler/handler.go

```diff
@@ -64,6 +64,77 @@ type Handler struct {
 	WebhookHandler         *webhooks.Handler
 }
 
+// @title PortainerCE API
+// @version 2.0.0
```

Comment:
it might be a future us problem, but this version is already out of date :)
Would be nice to centralise it

Reply:
I agree, swaggo gives an option for dynamic properties, but I think it's more for their internal swagger client, and not for the yaml. I'll check it

Reply:
I can't seem to find a way to change it dynamically. Fixed statically

Reply:
👍